tests/test_mssqlcliclient.py

```diff
@@ -66,51 +74,60 @@ def get_test_baseline(file_name):
         finally:
             mssql_cli_client.shutdown()
 
-    def test_connection(self):
+    @staticmethod
+    def test_connection():
         """
             Verify a successful connection via returned owner uri.
         """
         try:
             client = create_mssql_cli_client(owner_uri=u'connectionservicetest')
 
-            self.assertEqual(client.owner_uri, u'connectionservicetest')
-        finally:
-            shutdown(client)
-
-    def test_get_query_results(self):
-        """
-            Verify number of rows returned and returned query.
-        """
-        try:
-            client = create_mssql_cli_client()
-            test_query = u"""
-                select 1 as [ShiftID], 'Day' as [Name] UNION ALL
-                select 2, N'魚' UNION ALL
-                select 3, 'Night'
-            """
-
-            for rows, _, _, query, _ in client.execute_query(test_query):
-                self.assertTrue(len(rows), 3)
-                self.assertTrue(query, test_query)
+            assert client.owner_uri == u'connectionservicetest'
         finally:
             shutdown(client)
 
     @staticmethod
-    def test_json_writer_extra_params():
+    def test_json_writer_extra_params(client):
         """
             Verify JSON RPC accepts extra paramaters.
         """
         try:
-            client = create_mssql_cli_client()
             extra_params = client.extra_params
             json_writer = JsonRpcWriter(io.BytesIO())
             json_writer.send_request(u'test/method', extra_params, id=1)
         finally:
             json_writer.close()
-            shutdown(client)
```

Comment:
You'll see this removed in a few places--thanks to pytest fixtures, the test methods no longer need to handle client shutdown.